monkey/monkey_island/cc/services/zero_trust/zero_trust_service.py

```diff
@@ -1,20 +1,12 @@
-from typing import List
-
-from bson.objectid import ObjectId
-
-import common.data.zero_trust_consts as zero_trust_consts
+import common.common_consts.zero_trust_consts as zero_trust_consts
 from monkey_island.cc.models.zero_trust.finding import Finding
 
-# How many events of a single finding to return to UI.
-# 50 will return 50 latest and 50 oldest events from a finding
-EVENT_FETCH_CNT = 50
 
-
-class ZeroTrustService(object):
+class ZeroTrustService:
```

Comment:
I think what would make this class cleaner is the following:
* Add 2 new service files. 1 for pillars & 1 for principles. Move all functions from this file to one of the new files.
* Move this file, the 2 new files, `test_zero_trust_service.py`, `finding_service.py` and `monkey_details_service.py` to a new sub-package under this. I would name the package something like `report` but you can choose anything that makes sense.
* This file should have all the "exported" functions. The implementation should just be calling the appropriate function in any of the other files in this package.
* All the files/classes/functions in the new subpackage don't need a zerotrust prefix except for this one 
* `test_zero_trust_service.py` should be split accordingly to the split in `zero_trust_service.py`

Reply:
I refactored zero trust service into separate services. I also split security report and zero trust report endpoints. Now, when you're reading in `ZeroTrustReport` endpoint, I think this gives enough context to PillarService, FindingService and PrincipleService

Reply:
Perfect